app/controllers/form_controller.rb

```diff
@@ -321,4 +323,19 @@ def update_logs_with_duplicate_set_id(log, dynamic_duplicates, duplicate_set_id)
       duplicate.update!(duplicate_set_id: log.duplicate_set_id) if duplicate.duplicate_set_id != log.duplicate_set_id
     end
   end
+
+  def set_managing_organisation_to_owning_organisation?(owning_organisation)
+    return true if current_user.support? && @log.managing_organisation.blank? && owning_organisation&.managing_agents&.empty?
+    return true if owning_organisation&.absorbing_organisation == current_user.organisation
+    return true if @log.managing_organisation&.absorbing_organisation == current_user.organisation && owning_organisation == current_user.organisation
+
+    false
+  end
+
+  def set_managing_organisation_to_created_by_organisation?(saledate)
+    return false if current_user.support?
```

Comment:
I think there's a stupid case here where the original log creator was not support, but the person making the date change is and they don't think to also change the managing org. 

That's seems sufficiently unlikely to me that I assume we don't think it will happen.

Reply:
Yeah, I think you're right. I think it's probably unlikely, but let me double check if we want to do anything about it!

Reply:
This is sufficiently uncommon not to address it!